bin/test

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+cookiecutter \
+  --no-input \
+  --config-file bin/test-config.yml \
+  .
+
+cd ./TestProject
+
+bin/test
```

Comment:
Instead of renaming, what about this?

```sh
(cd ./TestProject && bin/test)
```

I feel like that more clearly shows the intent, and resolves the naming ambiguity.